media/css/privacy/firefox-privacy.scss

```diff
@@ -0,0 +1,282 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import
+    '../pebbles/includes/lib',
+    '../pebbles/includes/fonts/open-sans',
+    '../pebbles/includes/mixins/typography';
+
+* {
+    @include open-sans;
+}
+
+@font-face {
+    font-family: 'Open Sans';
+    font-weight: 600;
+    font-style: normal;
+    src: url('/media/fonts/opensans-semibold.woff2') format('woff2'),
+        url('/media/fonts/opensans-semibold.woff') format('woff');
+}
+
+$light-gray-background: #f2f2f2;
+
+.privacy-head,
+.privacy-lead-in,
+.privacy-footnote {
+    background-color: $light-gray-background;
+    color: #333;
+}
+
+/* POLICY HEADER */
+.privacy-head {
+    padding: 20px;
+    text-align: left;
+
+    @media #{$mq-desktop} {
+        position: sticky;
+        top: 0;
+        z-index: 2;
+
+        .privacy-head-wrapper {
+            margin: 0 auto;
+            padding-left: 17px;
+            max-width: 1420px;
+        }
+    }
+
+    h2 {
+        display: inline-block;
+        @include font-size-level3
+
+        @media #{$mq-desktop} {
+            margin: 0;
+        }
+
+        a {
+            color: inherit;
+            text-decoration: none;
+        }
+
+        &:before {
+            display: inline-block;
+            content: '';
+            @include at2x('/media/img/firefox/australis/fx38.0.5/firefox-logo.png', 30px, 33px);
```

Comment:
This is a logo that belongs to a different (and very old) page that could be removed at any time. I'd suggest using one from the shared /firefox/template/ directory instead.